.changeset/clean-mugs-wave.md

```diff
@@ -0,0 +1,26 @@
+---
+"@clerk/clerk-react": minor
+---
+
+Introducing experimental `asProvider`, `asStandalone` and `<X.Outlet/>` for UserButton and OrganizationSwitcher.
```

Comment:
```suggestion
Introducing experimental `asProvider`, `asStandalone`, and `<X.Outlet />` for `<UserButton />` and `<OrganizationSwitcher />` components.
```

Love me some [oxford comma](https://www.grammarly.com/blog/punctuation-capitalization/what-is-the-oxford-comma-and-why-do-people-care-so-much-about-it/)